.github/workflows/managed.yml

```diff
@@ -112,4 +112,4 @@ jobs:
           go env
           pwd
           git status
-          kubectl version --short --output json
+          kubectl version --output json
```

Comment:
We don't use `kubernetes` at all in this CI as I see, I guess we can remove this line completely.

Reply:
Done.